src/scripting/flash/display/flashdisplay.cpp

```diff
@@ -1008,10 +1008,7 @@ DisplayObject *DisplayObjectContainer::findRemovedLegacyChild(uint32_t name)
 bool Sprite::renderImpl(RenderContext& ctxt) const
 {
 	//Draw the dynamically added graphics, if any
-	bool ret = defaultRender(ctxt);
-
```

Comment:
IMHO this shouldn't be removed, because otherwise any drawing done directly on a sprite is not rendered

Reply:
I wasn't thinking about side effects, that's a good point.

I wasn't quite sure what the correct fix is, so I will remove this change.